clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -496,6 +496,15 @@ LogicalResult CastOp::verify() {
       return emitOpError() << "requires !cir.float type for result";
     return success();
   }
+  case cir::CastKind::addrspace_cast: {
```

Comment:
Do we need a test for verification? It's simple though.

Reply:
Yep, that'd be great!